### PR TITLE
Minor text edits for branding, punctuation, and TM symbol in title.

### PR DESCRIPTION
**Pull request by MCamp859**
Signed-off-by: MCamp859 <maryx.camp@intel.com>
@bd-dean  Please read over my text edits to be sure the Greengrass tutorial is still accurate. 
@krisdale @intelkevinputnam @mvincerx  I can't replicate the garbled ™ symbol in side nav, unclear if this PR will fix that issue.

### CODE REVIEW

File: source/clear-linux/tutorials/greengrass.rst [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
-Enable AWS Greengrass* and OpenVINO™ on |CL-ATTR|
-#################################################
-
-Hardware accelerated Function-as-a-Service (FaaS) enables cloud developers
-to deploy inference functionalities [1] on Intel® IoT edge devices with
-accelerators (CPU, Integrated GPU, Intel® FPGA, and Intel® Movidius™). These
-functions provide a great developer experience and seamless migration of
-visual analytics from cloud to edge in a secure manner using a containerized
-environment. Hardware-accelerated FaaS provides the best-in-class
+Enable AWS Greengrass\* and OpenVINO™ on |CL-ATTR|
```
Comment (krisdale) [inline source/clear-linux/tutorials/greengrass.rst]: Simplify page title to not use "Clear Linux OS" e.g. "Enable AWS Greengrass\* and OpenVINO™"

File: source/clear-linux/tutorials/greengrass.rst [hunk 2]
```diff
@@ -100,7 +101,7 @@ for the OpenVINO software stack:
 
    Learn more about how to :ref:`swupd-guide`.
 
-The ``computer-vision-basic`` bundle will install the OpenVINO software,
+The ``computer-vision-basic`` bundle installs the OpenVINO™ toolkit,
 along with the edge device models needed.
```
Comment (krisdale) [inline source/clear-linux/tutorials/greengrass.rst]: "... installs the OpenVINO™ toolkit and the sample models optimized for Intel® edge platforms"

File: source/clear-linux/tutorials/greengrass.rst [hunk 3]
```diff
@@ -100,7 +101,7 @@ for the OpenVINO software stack:
...
 
 Converting Deep Learning Models
```
Comment (krisdale) [inline source/clear-linux/tutorials/greengrass.rst]: Avoid words ending in -ing 😄 . Update section heading e.g. "Convert Deep Learning Models". Also, should this be sentence case? There are several section titles using -ing in this doc, so should do a scan of the section headers checking for correct casing and to convert from using -ing where possible.

File: source/clear-linux/tutorials/greengrass.rst [hunk 4]
```diff
@@ -109,8 +110,8 @@ Converting Deep Learning Models
 Locate Sample Models
```
Comment (krisdale) [inline source/clear-linux/tutorials/greengrass.rst]: Should this be sentence case?

File: source/clear-linux/tutorials/greengrass.rst [hunk 5]
```diff
@@ -126,7 +127,7 @@ with the Greengrass object detection sample.
 Running Model Optimizer
```
Comment (krisdale) [inline source/clear-linux/tutorials/greengrass.rst]: Use of -ing ...

**Comment by MCamp859** [comment]
@krisdale  Please have a look at the latest updates. 
@bd-dean  This tutorial has been significantly updated, please review when you get a chance.

File: source/clear-linux/tutorials/greengrass.rst [hunk 6]
```diff
@@ -282,20 +283,20 @@ configuring the lambda function for AWS Greengrass.
           (e.g. 1 for top-1 result, 5 for top-5 results)
 
 #. Add subscription to subscribe, or publish messages from AWS Greengrass
-   lambda function by following the steps 10-14 in `Configure the Lambda Function`_
+   lambda function by following the steps 10-14 in `Configure the Lambda Function`_.
```
[the remark this answers is not in the document]
Comment (MCamp859) [reply source/clear-linux/tutorials/greengrass.rst]: Removed some of the step numbers, where possible. Updated headings in our text to remove "-ing", note that some of the links match headings in the AWS docs.

**Comment by MCamp859** [comment]
@mvincerx   Ok to merge.

**Comment by mvincerx** [comment]
@MCamp859, Thank you. LGTM

**Pull request merged**